static/js/containers/UserPage_test.js

```diff
@@ -77,13 +78,64 @@ describe("UserPage", function() {
           getElementsByClassName('delete-button')[0];
       };
 
+      beforeEach(() => {
+        let userProfile = Object.assign({}, USER_PROFILE_RESPONSE, {
+          username: SETTINGS.username
+        });
+        userProfile.education.push({
+          "id": 3,
+          "degree_name": DOCTORATE,
+          "graduation_date": "2015-12-01",
+          "field_of_study": "Philosophy",
+          "school_name": "Harvard",
+          "school_city": "Cambridge",
+          "school_state_or_territory": "US-MA",
+          "school_country": "US",
+          "online_degree": false
+        });
+        helper.profileGetStub.
+          withArgs(SETTINGS.username).
+          returns(Promise.resolve(userProfile));
+      });
+
       it('shows the education component', () => {
         return renderComponent(`/users/${SETTINGS.username}`, userActions).then(([, div]) => {
           let title = div.getElementsByClassName('profile-card-title')[1];
           assert.equal(title.textContent, 'Education');
         });
       });
 
+      it('should show the entries in resume order', () => {
```

Comment:
- these two tests are very repetitious. basically identical structure with a few variables swapped here and there. wondering if there is something we can abstract here. 
- i'm seeing these tests for the first time, and it's not easy from the outset to figure out what they're actually doing. "education dialog index should be the first entry in sorted" is particularly vague. can you think of some comments to add/edit that would make it clearer to a first-timer?